openapi/components/schemas/Applications/ApplicationInstanceConfiguration.yaml

```
@@ -0,0 +1,37 @@
+type: object
+required:
+  - settings
+description: |-
+  Application instance configuration.
+
+  Only user can change settings when `configuredBy` is `user`.
+
+  Only the application itself can change settings when `configuredBy` is `application`.
+properties:
+  settings:
+    description: Settings that are available for the user to complete when they install the application.
```

suggestion: suggestion to help make the description more concise
```suggestion
    description: Configuration settings that are available during installation.
```